src/ctx.rs

```diff
@@ -625,13 +619,7 @@ macro_rules! into_ctx_float_impl {
             #[inline]
             fn into_ctx(self, dst: &mut [u8], le: Endian) {
                 assert!(dst.len() >= $size);
-                write_into!(
-                    signed_to_unsigned!($typ),
-                    $size,
-                    transmute::<$typ, signed_to_unsigned!($typ)>(self),
-                    dst,
-                    le
-                );
+                write_into!(signed_to_unsigned!($typ), $size, self.to_bits(), dst, le);
```

Comment:
some of this is rolling up fixes from the other PR that fixes unsafe stuff, i'm not sure it belongs in linting issue (unless clippy auto fixes this?)

Reply:
yes, this is what clippy fixes now